### PR TITLE
refactor, chore: (egen):sdk-automl-object-tracking-batch-prediction

### PR DESCRIPTION
**Pull request by k-root**
- REGION changed to LOCATION
- adds colab enterprise
- Removes future tense
- formats according to the template
- removes Boilerplate code

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb [hunk 1]
```diff
@@ -75,15 +80,15 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn to use `AutoML` to train a video model and use `Vertex AI Batch Prediction` to do batch predictions.\n",
+        "In this tutorial, you learn to use AutoML to train a video model and use Vertex AI Batch Prediction to do batch predictions.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb:83]: - change: "Vertex AI Batch Prediction"
- to: "Vertex AI batch prediction

File: notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb [hunk 2]
```diff
@@ -75,15 +80,15 @@
...
         "\n",
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `AutoML`\n",
-        "- `Vertex AI Batch Prediction`\n",
-        "- `Vertex AI Model` resource\n",
-        "- `Vertex AI Endpoint` resource\n",
+        "- AutoML\n",
+        "- Vertex AI Batch Prediction\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb:89]: - change: "Vertex AI Batch Prediction"
- to: "Vertex AI batch prediction

File: notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb [hunk 3]
```diff
@@ -488,10 +408,10 @@
       "source": [
         "### Create the Dataset\n",
         "\n",
-        "Next, create the `Dataset` resource using the `create` method for the `VideoDataset` class, which takes the following parameters:\n",
+        "Next, create the Dataset resource using the `create` method for the `VideoDataset` class, which takes the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb:411]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb [hunk 4]
```diff
@@ -488,10 +408,10 @@
...
         "\n",
-        "- `display_name`: The human readable name for the `Dataset` resource.\n",
-        "- `gcs_source`: A list of one or more dataset index files to import the data items into the `Dataset` resource.\n",
+        "- `display_name`: The human readable name for the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb:413]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb [hunk 5]
```diff
@@ -488,10 +408,10 @@
...
+        "- `gcs_source`: A list of one or more dataset index files to import the data items into the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-object-tracking-batch-prediction.ipynb:414]: - change: "Dataset resource"
- to: "dataset resource"

**Review by kittyabs — COMMENTED** [review]
numerous edits required

**Review by katiemn — APPROVED** [review]

**Pull request merged**